lib/tasks/gitlab/check.rake

```diff
@@ -278,6 +279,29 @@ namespace :gitlab do
       end
     end
 
+    def check_upload_perms
+      print "Upload directory permissions correct? ... "
+
+      upload_path = Rails.root.join(Gitlab.config.gitlab.user_home,"gitlab-rails","uploads")
+      upload_path_tmp = Rails.root.join(upload_path,"tmp")
+
+      # if tmp upload dir has incorrect permissions, assume others do as well
+      if File.stat(upload_path_tmp).mode == 16877 && File.owned?(upload_path_tmp) # verify drwxr-xr-x permissions
+        puts "yes".green
+      else
+        puts "no".red
+        try_fixing_it(
+          "sudo chown -R #{gitlab_user} #{upload_path}",
```

Comment:
What about non-omnibus though? Are you suggesting changing the ownership checking in this MR or just saying the way it is now is sufficient?